### ISSUE TITLE
Accecability problem in `<div role=region"><a class="skipToContent">` 

### ISSUE THREAD

**Issue by YoniChechik** [facebook/docusaurus#7981]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#reporting-new-issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have tried the `npm run clear` or `yarn clear` command.
- [X] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [X] I have tried creating a repro with https://new.docusaurus.io.
- [X] I have read the console error message carefully (if applicable).

### Description

missing aria-label in `<div role=region"><a class="skipToContent">` 

probebly should be `<div role=region" aria-label="skip To Content"><a class="skipToContent">` 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role

### Steps to reproduce

above

### Expected behavior

above

### Actual behavior

above

### Your environment

happens in all docusaurus sites

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by Josh-Cena** [comment]
I think this makes sense. Would you want to send a PR?

**Comment by YoniChechik** [comment]
@Josh-Cena PR sent :)
